### DISCUSSION TITLE
[Kavita] Idea / Being able to set side-padding regardless of reader type

### DISCUSSION THREAD

**Discussion by eswiig** [Kareadita/Kavita#2872] [Ideas]
### Idea Description

It hurts my eyes if I read full width and if I restrict by height, it becomes too small for me to read.

I would like to be able to set side padding regardless of if I am reading manga, images, books or pdfs.

![image](https://github.com/Kareadita/Kavita/assets/58608430/37c12d77-4d38-40e3-88bc-c429f6290578)

### Idea Category

Feature Enhancement

### Duration of Using Kavita

1 hour, been using other software, but want to try using Kavita more

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
This is a duplicate of https://github.com/Kareadita/Kavita/discussions/2558
